<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostStreamWindow</class>
 <widget class="QDialog" name="HostStreamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: rgb(54, 120, 156);
	color: rgb(223, 234, 237);
	border: 2px solid rgb(160, 207, 226);
	min-height: 40px;
	min-width: 40px;
	border-radius: 20px;
}
QPushButton:pressed {
	background-color: rgba(173, 213, 255, 80);
}
QPushButton#hostButton {
	background-color: rgb(217, 162, 64);
	border: 2px solid rgb(249, 220, 161);
}
QPushButton#hostButton:pressed {
	background-color: rgba(217, 162, 64, 80);
	border: 2px solid rgb(249, 220, 161);
}

QLabel {
	color: rgb(223, 234, 237);
}
QLabel[header=true] {
	font-size: 30px;
}

QDialog#HostStreamWindow {
	background-color: rgb(11, 17, 20);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Host Stream</string>
     </property>
     <property name="header" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Display Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Stream Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="streamNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ip_address_button">
     <property name="text">
      <string>127.0.22.45</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::EditCopy"/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hostButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
